lib/user_data_parser/file/file_parser.rb

```diff
@@ -0,0 +1,23 @@
+require_relative "../../../spec/config/test_config.rb"
+require_relative "../formatter.rb"
+require_relative "../row/row_parser.rb"
+require_relative "../data_set/data_set_parser.rb"
+
+module UserDataParser
```

Comment:
Starting to get lost in the similarity of the names. I'm already seeing ```UserDataParser```, ```FileParser```, and ```DataSetParser```--when I read those, they all sound like they do the same thing. Can you think of a way to get more clarity out of the naming convention?